helm/frost-server/templates/mqtt-deployment.yaml

```diff
@@ -62,17 +62,17 @@ spec:
             - name: mqtt_mqttServerImplementationClass
               value: "de.fraunhofer.iosb.ilt.sensorthingsserver.mqtt.moquette.MoquetteMqttServer"
             - name: mqtt_Enabled
-              value: "true"
+              value: "{{ .Values.frost.mqtt.enabled }}"
             - name: mqtt_Port
-              value: "1883"
+              value: "{{ .Values.frost.mqtt.ports.mqtt.servicePort }}"
             - name: mqtt_Host
-              value: "0.0.0.0"
+              value: "{{ .Values.frost.mqtt.host }}"
             - name: mqtt_internalHost
-              value: "localhost"
+              value: "{{ .Values.frost.mqtt.internalHost }}"
             - name: mqtt_WebsocketPort
-              value: "9876"
```

Comment:
Using `Values.frost.mqtt.ports.websocket.servicePort` is not correct, best leave the ports alone for now.